pkg/volume/plugins.go

```diff
@@ -743,3 +744,21 @@ type dummyPluginProber struct{}
 
 func (*dummyPluginProber) Init() error                          { return nil }
 func (*dummyPluginProber) Probe() (bool, []VolumePlugin, error) { return false, nil, nil }
+
+// getBusyboxImage returns multi-arch busybox docker image
+func getBusyBoxImage() string {
```

Comment:
Thanks @dixudx , but this may not work in offline mode, as I have to first download the image first for all platform and put it to all hosts.

I would suggest we treat this as another container runtime parameter like what we did for default pod sandbox, please refer to https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/config/flags.go#L84 for detail.

Reply:
> but this may not work in offline mode, as I have to first download the image first for all platform and put it to all hosts.

If working in offline mode, you still need to download all `gcr.io/kubernetes/pause` images for all platforms. Just one more image will not add too much trouble, right?

> I would suggest we treat this as another container runtime parameter like what we did for default pod sandbox, please refer to https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/config/flags.go#L84 for detail.

@gyliu513 As `pause` image is the core part of a pod, a flag like this is reasonable. But for `busybox`, it is only used by pod `"pv-recycler"`, we may not need introducing such a new flag that will increase the complexity to start `kubelet`. And users may get confused and misconfigure this value. WDYT?

Keep open on adding such a flag if needed.

Reply:
> If working in offline mode, you still need to download all gcr.io/kubernetes/pause images for all platforms. Just one more image will not add too much trouble, right?

It will become a limitation of kubernetes if we do not support this. If we can support configuring recycle image, then I can just put the recycle image to a private registry and then configure kubelet use recycler image from the registry, this is more simple to maintain.

If keep it hard code in kubernetes code, we may need to document how to do this in offline mode which may bring some maintain effort for the customer.

Let's wait some comments from others ;-)

Reply:
> If we can support configuring recycle image, then I can just put the recycle image to a private registry and then configure kubelet use recycler image from the registry, this is more simple to maintain.

Make sense. If the registry supports [image manifest v2](https://docs.docker.com/registry/spec/manifest-v2-2/), then a simple string `busybox` is enough since multi-arch is supported already. If not, like `gcr.io`, we still need to determine the arch to choose the right busybox image here, which is also for backward compatible consideration.

> If keep it hard code in kubernetes code, we may need to document how to do this in offline mode which may bring some maintain effort for the customer.

Right. Agree.